app/models/answer.rb

```diff
@@ -8,6 +8,12 @@ class Answer < ApplicationRecord
   has_many :subscriptions, dependent: :destroy
   has_many :comment_smiles, through: :comments, source: :smiles
 
+  # rubocop:disable Rails/UniqueValidationWithoutIndex
+  # This cop is disabled here because there already are questions with
+  # multiple answers. Adding an index would break things database-side
```

Comment:
We could probably come up with a migration that merges duplicate answers and adds the index.  For now this is good enough